prime-dotnet-webapi/Services/Rules/MinorUpdateRules.cs

```diff
@@ -118,6 +118,7 @@ public override Task<bool> ProcessRule(Enrollee enrollee)
                 return Task.FromResult(false);
             }
 
+
```

Comment:
Don't really need this new line, I assume this was left over from deleting some code you added here?

Reply:
Removed